### PR TITLE
VL-S35-Header

### PR DESCRIPTION
**Pull request by doolaik**
Header integrated along with background coloring

### CODE REVIEW

File: src/Screens/Home/HomeScreen.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { Text, ScrollView } from "react-native";
 import React from "react";
 import styles from "../../Shared/styles.js";
 import HomeScreenButton from "../../Shared/Components/HomeScreenButton.js";
-
+import Breadcrumb from "../../Shared/Components/header/Breadcrumb";
```
Comment (PizzaCat07) [inline src/Screens/Home/HomeScreen.js:5]: Breadcrumb imported not used

File: src/Shared/Components/header/Breadcrumb.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from "react";
+import { View, Text, Pressable } from "react-native";
+import { useNavigation, useRoute } from "@react-navigation/native";
+import MaterialIcons from "react-native-vector-icons/MaterialIcons";
+
+import styles from "../../styles";
+
+const Breadcrumb = ({ partLabel }) => {
```
Comment (PizzaCat07) [inline src/Shared/Components/header/Breadcrumb.js:8]: Needs comments on where props are coming from and/or what the mean

File: src/Shared/Components/header/CustomHeader.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+// Because React Native does not support the outline style prop, we've had to manually simulate the outline
+//    function by using the borderWidth and making it "on" or "off" with the isFocused state. To achieve this,
+//    container has a paddingTop of 10, and the logoPadding has a padding of 3 giving a total paddingTop of 13.
+//    The logoPadding of 3 is required as the Searchbar has a borderWidth of 3 and without the padding and
+//    additional height on that container (56, with 3 on top and bottom), it makes the logo offset from the
+//    searchbar by 3. The same had to be done with the topRowSpacing.
+
+// Logo can take the logoHeight and logoWidth arguments. By default, it will be H400 x W416 (full size image)
+
+export const CustomHeader = ({ navigation, options, back, partLabel }) => {
```
Comment (PizzaCat07) [inline src/Shared/Components/header/CustomHeader.js:22]: need comments on what props are going in, where they are from.

File: src/Shared/Navigation/MainStack.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
```
Comment (PizzaCat07) [inline src/Shared/Navigation/MainStack.js:1]: Is this stack still needed ? did a serach and it did not come up. detele if not needed

File: src/Screens/Home/HomeScreen.js (hunk 1 above)
Comment (doolaik) [reply src/Screens/Home/HomeScreen.js:5]: Breadcrumb now implemented.

**Review by PizzaCat07 — CHANGES_REQUESTED** [review]

File: src/Shared/Components/header/Breadcrumb.js (hunk 2 above)
Comment (doolaik) [reply src/Shared/Components/header/Breadcrumb.js:8]: Documentation added.
Changed to custom icon component.

File: src/Shared/Components/header/CustomHeader.js (hunk 3 above)
Comment (doolaik) [reply src/Shared/Components/header/CustomHeader.js:22]: All props removed; they were unneeded.

File: src/Shared/Navigation/MainStack.js (hunk 4 above)
Comment (doolaik) [reply src/Shared/Navigation/MainStack.js]: Removed

**Review by PizzaCat07 — APPROVED** [review]

**Pull request merged**